compiler/importer.nim

```diff
@@ -335,7 +336,7 @@ proc impMod(c: PContext; it: PNode; importStmtResult: PNode) =
   let m = myImportModule(c, it, importStmtResult)
   if m != nil:
     # ``addDecl`` needs to be done before ``importAllSymbols``!
-    addDecl(c, m, it.info) # add symbol to symbol table of module
+    addDecl(c, m) # add symbol to symbol table of module
```

Comment:
Seems better to use `addDecl(c, m, m.info)` here and elsewhere and keep the line information explicit in the data model.

Reply:
The `addDecl` overload here gets the line info from `m`.
Or do you more mean get the line info here so its more clear that its coming from `m`?